internal/rbd/rbd_journal.go

```diff
@@ -587,6 +587,10 @@ func RegenerateJournal(
 	if rbdVol.JournalPool == "" {
 		rbdVol.JournalPool = rbdVol.Pool
 	}
+	if rbdVol.Owner, ok = volumeAttributes["csi.storage.k8s.io/pvc/namespace"]; !ok && kmsID != "" {
+		util.DebugLog(ctx, "could not detect owner for %s", rbdVol.String())
```

Comment:
nit: This should move to a helper function that gets shared with `initKMS()`, copy/pasting parts of code is practically never a good thing to so. This is Kubernetes specific, so you could place the helper in `internal/util/k8s.go`.

`rbdVol` is a pointer to a `rbdVolume`, so there is no need to use `.String()`.